tests/sql/getConnectionParams.test.js

```diff
@@ -107,15 +104,14 @@ describe('getConnectionParams', () => {
   });
 
   it('Fails if there is no writer endpoint available', async () => {
-    config.clusterEnv = 'staging';
     config.awsRegion = 'eu-west-1';
```

Comment:
afaict the test config already has `awsRegion` correctly set so this shouldn't be needed